app/components/TrackCard/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ *
+ * TrackCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card } from 'antd';
+import { PlayCircleTwoTone, RightCircleTwoTone, PauseCircleTwoTone } from '@ant-design/icons';
+import * as colors from '@app/themes/colors';
+import If from '@components/If';
+
+const { Meta } = Card;
+
+const CustomCard = styled(Card)`
+  && {
+    width: 18em;
+    margin: 1rem 0;
+    height: ${(props) => (props.height ? props.height : 30)}em;
+    background-color: ${colors.TrackCardBColor};
+  }
+`;
+const TextCard = styled(Meta)`
+  && {
+    margin: 0.2rem !important;
+  }
+`;
+const Image = styled.img`
+  width: 250px;
+  height: 18em;
+  magin-bottom: 2em;
```

Comment:
Just a nit. There are methods in styles.js for many of these. Consider using those just to get a feeling of them.